clvm/SExp.py

```diff
@@ -112,7 +112,7 @@ def to(class_, v: CastableType):
         return class_(v1)
 
     def cons(self, right: "CLVMObject"):
```

Comment:
as I understand it, `right` will always be  CLVM object here, and can be passed directly into the right hand side of the cons-box